lib/krb5/krbhst.c

```diff
@@ -353,14 +353,15 @@ krb5_krbhst_format_string(krb5_context context, const krb5_krbhst_info *host,
 			  char *hostname, size_t hostlen)
 {
     const char *proto = "";
-    char portstr[7] = "";
     if(host->proto == KRB5_KRBHST_TCP)
 	proto = "tcp/";
     else if(host->proto == KRB5_KRBHST_HTTP)
 	proto = "http://";
-    if(host->port != host->def_port)
-	snprintf(portstr, sizeof(portstr), ":%d", host->port);
-    snprintf(hostname, hostlen, "%s%s%s", proto, host->hostname, portstr);
+    if(host->port != host->def_port) {
```

Comment:
Although there are lots of instances of `if()` in the tree instead of `if ()` our preferred style is a space after `if` as `if` is not a function.   We are not going to globally fix this because we do not want to destroy the usefulness of git blame.

Reply:
Thanks, is there a document on this somewhere?  

(This is Samba's style also, for what it is worth)